build/workshops.md

```diff
@@ -2,8 +2,8 @@
 
 Contract proxy kit and Safe apps as well as the Safe apps SDK have been presented at several events.
```

Comment:
This paragraph should be updated as it is a bit empty in content. Mentioning the CPK should be removed from here as it is not relevant nowadays

Reply:
Removed: https://github.com/safe-global/safe-docs/pull/196/commits/18421d583424821dbfa6aab3c673087a96d58dc1